web/src/helpers/constants.js

```diff
@@ -41,7 +41,7 @@ const LANGUAGE_NAMES = {
   da: 'Dansk',
   de: 'Deutsch',
   el: 'Ελληνικά',
-  en: 'English',
```

Comment:
I'm not sure this change is necessary but if we decide to sitck with it we should add the geography tag to all of the languages.

Reply:
i18n changes the local storages with these constants ```en-GB, zh-TW, zh-CN, zh-HK``` when any of these languages are selected. 
If these are not changed, then it would not be mapped to right language constant, and it would not be selected in the language menu.

Reply:
I guess the only issue is we have a failed network request for en-GB then it falls back to en.

![image](https://user-images.githubusercontent.com/18622768/193585046-db16df96-bc8e-42ef-9940-5da05893866e.png)

Reply:
If we decide to add locale tags here we need to update the i18n fallback language to match.

With that said I'm not totally sure we should do this, at least not for English until we have a US English version and a British English version as it will also affect all number and date formating.